src/main/java/com/openlattice/shuttle/transforms/HashTransform.java

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
This class should stay in shuttle b/c it's a concrete Transformation. I think a small refactor to the `Builder` in `PropertyDefinition` will let you move it back

Reply:
Idk if it will be small...

Reply:
🤔 
Isn't it only referenced in the `PropertyDefinition.Builder` that calls through to a more generic method that takes `List<Transformation>` and only used by a convenience method that is called by tests? 
You should be able to just create a HashTransform in that test and pass it through the more generic method.

Reply:
It might be called by people's local scripts to do integrations though?